test/Microsoft.TestPlatform.AcceptanceTests/ResultsDirectoryTests.cs

```diff
@@ -1,13 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
```

Comment:
Changes to this file are unrelated to this PR but fixes the failing acceptance tests that were blocking this PR.